test/utils/trends/data.test.js

```diff
@@ -49,12 +49,12 @@ describe('[trends] data utils', () => {
         expect(utils.findTimeOfDayBin(binSize, 0)).to.equal(1800000);
       });
 
-      it('should assign a bin of `1800000` to a datum at time 3600000', () => {
-        expect(utils.findTimeOfDayBin(binSize, 3600000)).to.equal(1800000);
+      it('should assign a bin of `1800000` to a datum at time 3599999', () => {
+        expect(utils.findTimeOfDayBin(binSize, 3599999)).to.equal(1800000);
```

Comment:
The bins are times - the time at the center of each slice. Also the function is meant to be read: find time-of-day bin (e.g., find bin for this particular time of day), not find time of day-bin or something like that. The return is supposed to be a bin, not a time. But the bins are times anyway :P